stac_fastapi/elasticsearch/stac_fastapi/elasticsearch/database_logic.py

```diff
@@ -860,22 +822,27 @@ async def get_all_catalogs(
         catalog_indices_list = []
         for hit in hits:
             # Construct required catalog indices
-            catalog_index = hit["_index"].split("_", 1)[1]
+            catalog_index = hit["_index"].split("_", 1)
```

Comment:
Would there be any advantage to moving this inside the `try` and combining it with line 828?

Reply:
Good suggestion, moved into the `try-except` block